src/mongo/db/curop.cpp

```diff
@@ -446,6 +446,8 @@ uint64_t CurOp::MaxTimeTracker::getRemainingMicros() const {
     return _targetEpochMicros - now;
 }
 
+PseudoRandom CurOp::_prng(std::unique_ptr<SecureRandom>(SecureRandom::create())->nextInt64());
```

Comment:
Just creating:
`static PseudoRandom _prng(static_cast<int64_t>(time(0)));`

is enough. We don't need `SecureRandom` as it's slower and should be used for crypto.